programs/bpf_loader/src/syscalls.rs

```diff
@@ -48,6 +49,8 @@ pub enum SyscallError {
     ProgramNotSupported,
     #[error("{0}")]
     InstructionError(InstructionError),
+    #[error("Piviledge escalation in cross-program invocation")]
+    PriviledgeEscalation,
```

Comment:
Seems like users would be confused by this error. Maybe "unauthorized something"?

Reply:
Also, Piviledge/Priviledge => Privilege

Reply:
Heh, I'll rework it, sounds so cool though!

Reply:
Thanks @CriesofCarrots 

Reworked...

Reply:
Just do a quick search for "priviledge". Looks like there are 14 other instances (const names, and one `info`)

Reply:
Heh, thanks @CriesofCarrots, I think I'm spelling it as I say it ;-)